site/src/components/WorkspaceScheduleForm/WorkspaceScheduleForm.tsx

```diff
@@ -48,15 +50,15 @@ export const Language = {
   errorTtlMax:
     "Please enter a limit that is less than or equal to 168 hours (7 days).",
   daysOfWeekLabel: "Days of Week",
-  daySundayLabel: "Sunday",
-  dayMondayLabel: "Monday",
-  dayTuesdayLabel: "Tuesday",
-  dayWednesdayLabel: "Wednesday",
-  dayThursdayLabel: "Thursday",
-  dayFridayLabel: "Friday",
-  daySaturdayLabel: "Saturday",
+  daySundayLabel: "Sun",
+  dayMondayLabel: "Mon",
+  dayTuesdayLabel: "Tue",
+  dayWednesdayLabel: "Wed",
+  dayThursdayLabel: "Thu",
+  dayFridayLabel: "Fri",
+  daySaturdayLabel: "Sat",
   startTimeLabel: "Start time",
-  startTimeHelperText: "Your workspace will automatically start at this time.",
+  startTimeHelperText: "Time the workspace will automatically start.",
```

Comment:
I think the old string is a little more grammatical - is there a reason we changed it?

Reply:
Maybe we don't need this helper text at all anymore considering we have a label and also a general field description.

Reply:
That is true, what do you think is best?

Reply:
I would just take it out!